toonz/sources/toonz/xshcellviewer.cpp

```diff
@@ -1042,6 +1049,16 @@ void CellArea::drawLevelCell(QPainter &p, int row, int col, bool isReference) {
   TXshCell cell = xsh->getCell(row, col);
   TXshCell prevCell;
 
+  // If the user change the option to start a sequence with 0 to 1 in-live, we
+  // delete all cell and frame 0 of the actual Xsheet
+  if (!Preferences::instance()->isSequenceCanStartWith0() &&
+      cell.getFrameId().getNumber() == 0) {
+    cell.getSimpleLevel()->eraseFrame(TFrameId(0));
```

Comment:
> do you see accesible only closing and opening the actual scene programatically?

Yes, I agree that just closing and start a new scene would work. I think you can do it by triggering "New Scene" command like as follows:
`CommandManager::instance()->getAction("MI_NewScene")->trigger();`

I realized that allowing frame 0 will have influence over more factors in the software than expected. Thank you very much for tackling this and your contribution!